syntax/table.php

```diff
@@ -391,6 +391,19 @@ function preList($clist, $data) {
                 '</div>';
             $text .= '</div>';
         }
+        
+        // fixes for bootstrap tpl 
+        if( isset($conf['tpl']['bootstrap3']['tableStyle'])) {
+            $ts = explode(',', $conf['tpl']['bootstrap3']['tableStyle']);
+            foreach ($ts as $class) {
+                if ($class == 'responsive') {
+                    $text = str_replace('<div class="table', '<div class="table table-responsive', $text);
```

Comment:
What about adding this as if at line 370? Than the str_replace() is not needed, it is?
And further, is a tpl specific solution here the best?